speakers/speaker-recruitment.md

```diff
@@ -0,0 +1,142 @@
+
+With a well crafted Call for Proposals, it is important to get the word out to the audiences you want to target. Having an organizing committee that can tap into their different networks helps. If the conference is attended by a diverse population, it can be useful to encourage those attendees to apply to speak.  Advertising the opportunity to speak is about getting the word out to the right populations. Social media networks like Twitter, LinkedIn, and Facebook have communities focused on promoting speaker and leadership opportunities. Use these Diversity, People of Color, and Women in Tech based resources to spread the word about the opportunities offered by your conference. You can also reach out to meetups in your region to promote the event and find new speakers who are looking for a chance to shine. While adverstising the call, don't be afraid to ask for help getting the word out that the conference is looking for underrepresented speakers. People notice when a conference is not talking about diversity and when a conference is. Letting the community know that your conference is talking about it will help attract the speakers you are looking for. Also, don't be afraid to ask for recommendations for speakers or approach individuals about proposing a talk. This type of personal connection can help inform people about opportunities and show them that their participation is valued.
+
+#### Key points:
+
+  * Be clear about the range of talk topics and formats that you encourage. Don’t assume that people know “the usual drill” or are confident enough to apply, despite a perceived lack of fit.
+  * Remove uncertainty around the selection criteria and the resources available to speakers. Can you share your evaluation rubric or process? Describe, e.g., fee waivers, travel reimbursement, support for child care or remote presentations.
+  * Use all available methods to get your Call for Proposals in front of a large and diverse audience. There is an abundance of networks and social media communities that exist exactly for this purpose.
+
+#### Links to networks:
+
+  1. LatinX in AI: http://www.latinxinai.org 
+    a. Facebook: https://www.facebook.com/latinxinai/ 
+    b. Twitter: https://twitter.com/_LXAI 
```

Comment:
I _think_ the `_` is making all the remaining lines show up in italic, and I think you can fix that by doing `<https://twitter.com/_LXAI>`

Reply:
I added the surrounding angle brackets, but the github syntax highlighting for the "Files changed" part of this PR continues to show the remaining lines in italics. I tried it in a gist, and using the angle brackets around the url did seem fix the italics text in that editor.

The rendered markdown, seen at https://github.com/RConsortium/RCDI-WG/blob/58105bfadb19a9e16adf4a7c8b35f3d37b5cf36b/speakers/speaker-recruitment.md, doesn't show the italics. Hopefully the merged version also won't show the italics.

I also adjusted the spacing in some of the lists to make sure the markdown renders with properly indented paragraphs.